include/shellstrings.h

```diff
@@ -41,7 +53,18 @@ void shell_intro();
  * Returns:
  *  - None.
  */
-void shell_save();
+void shell_save(bool *color);
```

Comment:
does color have to be passed in as a parameter for all these functions? can't we just declare a global flag at the top of the file?

Reply:
no, because the output is determined by user.